lib/Runtime/ByteCode/ByteCodeDumper.cpp

```diff
@@ -27,12 +27,6 @@ namespace Js
     }
     void ByteCodeDumper::Dump(FunctionBody* dumpFunction)
     {
-        if (!CONFIG_FLAG(DumpDbgControllerBytecode) && dumpFunction->GetSourceContextInfo() &&
```

Comment:
I didn't know there was special handling for test file. Good to get rid of that.